projects/public-search/src/app/person-brief/person-brief.component.html

```diff
@@ -16,7 +16,7 @@
 -->
 <div *ngIf="record">
 <h5 class="card-title mb-0 rero-ils-person">
-  <a href="{{ detailUrl }}">{{record.metadata | mefTitle}}</a>
+  <a href="{{ detailUrl.link }}">{{record.metadata | mefTitle}}</a>
```

Comment:
Remove href tag and add `[routerLink]="[detailUrl.link]"`

Reply:
Done